### ISSUE TITLE
Request for clearer error messages/fail safe system

### ISSUE THREAD

**Issue by Prunt** [NuVotifier/NuVotifier#25]
First of all, this is an awesome plugin and I hope you keep working on it.

I've been using NuVotifier with pluginMessaging this far, but since you drop support for it, I decided to switch to the recommended proxy forwarding. So, I spent about an hour figuring out why the hell votes wouldn't forward from BungeeCord server to the Spigot instance. I looked through every tutorial I could find to spot the problem. I finally got it working when I removed unused servers from the proxy servers list, because - apparently - the plugin stops working when there's a server in the list which doesn't have a token defined.

The only error I had in my console was this:
```
01:24:29 [WARNING] Exception encountered when loading plugin: NuVotifier
java.lang.IllegalArgumentException: Empty key
<------>at javax.crypto.spec.SecretKeySpec.<init>(SecretKeySpec.java:94)
<------>at com.vexsoftware.votifier.util.KeyCreator.createKeyFrom(KeyCreator.java:9)
<------>at com.vexsoftware.votifier.bungee.NuVotifier.onEnable(NuVotifier.java:297)
<------>at net.md_5.bungee.api.plugin.PluginManager.enablePlugins(PluginManager.java:227)
<------>at net.md_5.bungee.BungeeCord.start(BungeeCord.java:273)
<------>at net.md_5.bungee.BungeeCordLauncher.main(BungeeCordLauncher.java:55)
<------>at net.md_5.bungee.Bootstrap.main(Bootstrap.java:15)
```
At first I didn't make much of this, I just thought it was because of the debug mode I enabled, but later I realised what it meant.

Long story short, if you could implement a fail safe system (just skip invalid server sections) or a better explained error messages, it could save a lot of time for someone else.

Cheers!

**Comment by Hodel1** [comment]
i have the same issue. i will test it within next day. When it works, i do a pull request.
